CrossAppLoginUI/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id("com.android.library")
+    alias(core.plugins.kotlin.android)
+    alias(core.plugins.compose.compiler)
+    kotlin("plugin.parcelize")
+    kotlin("plugin.serialization")
+    id("androidx.navigation.safeargs.kotlin")
+}
+
+val coreCompileSdk: Int by rootProject.extra
+val coreMinSdk: Int by rootProject.extra
+val javaVersion: JavaVersion by rootProject.extra
+
+android {
+
+    namespace = "com.infomaniak.core.crossloginui"
```

Comment:
Move this module as a submodule of CrossAppLogin.

Reply:
Will be done later, it's not a blocker.